2020/04/29 13:21:27 [crit] 8339#8339: *195 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /mwo/order/create/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/mwo/order/create/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/29 13:21:28 [crit] 8339#8339: *195 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /mwo/order/create/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/mwo/order/create/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/29 13:21:31 [crit] 8339#8339: *195 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /mwo/order-list/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/mwo/order-list/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order/create/"
2020/04/29 13:21:31 [crit] 8339#8339: *195 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /mwo/order/create/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/mwo/order/create/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/29 13:22:22 [crit] 8339#8339: *195 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /mwo/order/create/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/mwo/order/create/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 07:57:16 [error] 7764#7764: *2 open() "/home/vadim/python_projects/rahco/static/static/css/hyper.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/hyper.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 07:57:16 [error] 7764#7764: *2 open() "/home/vadim/python_projects/rahco/static/static/css/custom.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/custom.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 07:57:20 [error] 7764#7764: *2 open() "/home/vadim/python_projects/rahco/static/static/css/hyper.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/hyper.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/"
2020/04/30 07:57:20 [error] 7764#7764: *2 open() "/home/vadim/python_projects/rahco/static/static/css/custom.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/custom.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/"
2020/04/30 07:58:29 [error] 7887#7887: *2 open() "/home/vadim/python_projects/rahco/static/static/css/hyper.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/hyper.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/"
2020/04/30 07:58:29 [error] 7887#7887: *2 open() "/home/vadim/python_projects/rahco/static/static/css/custom.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/custom.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/"
2020/04/30 07:58:33 [error] 7887#7887: *2 open() "/home/vadim/python_projects/rahco/static/static/css/hyper.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/hyper.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/"
2020/04/30 07:58:33 [error] 7887#7887: *2 open() "/home/vadim/python_projects/rahco/static/static/css/custom.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/custom.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/"
2020/04/30 07:58:39 [error] 7887#7887: *2 open() "/home/vadim/python_projects/rahco/static/static/css/hyper.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/hyper.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 07:58:39 [error] 7887#7887: *2 open() "/home/vadim/python_projects/rahco/static/static/css/custom.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/custom.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 07:59:05 [error] 7941#7941: *3 open() "/home/vadim/python_projects/rahco/static/static/css/hyper.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/hyper.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 07:59:05 [error] 7941#7941: *3 open() "/home/vadim/python_projects/rahco/static/static/css/custom.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/custom.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 07:59:06 [error] 7941#7941: *3 open() "/home/vadim/python_projects/rahco/static/static/css/hyper.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/hyper.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 07:59:06 [error] 7941#7941: *3 open() "/home/vadim/python_projects/rahco/static/static/css/custom.css" failed (2: No such file or directory), client: 192.168.138.195, server: , request: "GET /static/css/custom.css HTTP/2.0", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 08:06:04 [crit] 8605#8605: *2 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /inventory/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/inventory/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 08:06:05 [crit] 8605#8605: *2 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /inventory/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/inventory/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 08:11:53 [crit] 9171#9171: *2 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /inventory/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/inventory/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 08:11:55 [crit] 9171#9171: *2 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /inventory/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/inventory/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 08:11:55 [crit] 9171#9171: *2 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET /inventory/ HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/inventory/", host: "192.168.138.195", referrer: "https://192.168.138.195/mwo/order-list/"
2020/04/30 08:12:08 [crit] 9171#9171: *2 connect() to unix:/home/vadim/python_projects/rahco/erp/run/erp.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.138.195, server: , request: "GET / HTTP/2.0", upstream: "http://unix:/home/vadim/python_projects/rahco/erp/run/erp.sock:/", host: "192.168.138.195"
